build/Microsoft.DotNet.Cli.Prepare.targets

```diff
@@ -29,11 +29,17 @@
       <DotNetPath>$(Stage0Path)</DotNetPath>
 
       <BaseOutputDirectory>$(RepoRoot)/artifacts/$(Rid)</BaseOutputDirectory>
-      <OutputDirectory>$(BaseOutputDirectory)/stage2/</OutputDirectory>
+      <ToolsOutputDirectory>$(BaseOutputDirectory)/tools</ToolsOutputDirectory>
+      <Stage1Directory>$(BaseOutputDirectory)/stage1</Stage1Directory>
+      <Stage1SymbolsDirectory>$(BaseOutputDirectory)/stage1symbols</Stage1SymbolsDirectory>
+      <DotnetStage1>$(Stage1Directory)/dotnet.exe</DotnetStage1>
+      <Stage2Directory>$(BaseOutputDirectory)/stage2</Stage2Directory>
```

Comment:
@brthor just fixed this to have a trailing slash.  Your change is undoing what he did.